### PR TITLE
feat: HTML Input and Images (#40324)

### PR DESCRIPTION
**Pull request by Ankush-Negi**

### CODE REVIEW

File: html/css/style.css [hunk 1]
```diff
@@ -41,12 +54,66 @@
...
     padding: 1%;
     flex-basis: 40%;
     background-color: #efeef0;
+    display: flex;
+    flex-direction: column;
+}
+
+.first__wallet {
+    padding-top: 1%;
+    padding-bottom: 1%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: can use only padding instead of these both and give 0 for left and right

File: html/css/style.css [hunk 2]
```diff
@@ -41,12 +54,66 @@
...
+    border-bottom: solid;
+    border-width: thin;
+    border-color: lightgray;
+}
+
+.first__cardDetails {
+    padding-top: 2%;
+    padding-left: 25%;
+    padding-right: 25%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: Make this into single padding

File: html/css/style.css [hunk 3]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
     padding: 1%;
     content: "/";
 }
+
+input[type=text1],
+select {
+    width: 100%;
+    padding: 2%;
+    margin-top: 4%;
+    margin-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 4]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    border: 1px solid #ccc;
+    border-radius: 4px;
+}
+
+input[type=text2],
+select {
+    width: 100%;
+    padding: 2%;
+    margin-top: 2%;
+    margin-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 5]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    border-radius: 4px;
+}
+
+input[type=text3],
+select {
+    width: 47%;
+    padding: 2%;
+    margin-top: 2%;
+    margin-right: 4%;
+    margin-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 6]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    border: 1px solid #ccc;
+    border-radius: 4px;
+}
+
+input[type=text4],
+select {
+    width: 47%;
+    padding: 2%;
+    margin-top: 2%;
+    margin-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 7]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+}
+
+.second__product {
+    border-bottom: solid;
+    border-width: thin;
+    border-color: lightgray;
+    display: flex;
+    flex-direction: row;
+    padding-top: 2%;
+    padding-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 8]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+}
+
+.second__product--lastProduct {
+    border-bottom: solid;
+    border-width: thin;
+    border-color: lightgray;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 9]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    display: flex;
+    flex-direction: row;
+    padding-top: 2%;
+    padding-bottom: 30%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 10]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    padding: 1%;
+    flex-basis: 15%;
+}
+
+.second__pricing{
+    display: flex;
+    flex-direction: column;
+    border-bottom: solid;
+    border-width: thin;
+    border-color: lightgray;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css [hunk 11]
```diff
@@ -58,3 +125,121 @@ ul.breadcrumb li+li:before {
...
+    padding-top: 2%;
+    padding-bottom: 2%;
```
Comment (VisaaliSuccessive) [inline html/css/style.css]: make it single

File: html/css/style.css (hunk 1 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 2 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 3 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 4 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 5 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 6 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 7 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 8 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 9 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 10 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

File: html/css/style.css (hunk 11 above)
Comment (Ankush-Negi) [reply html/css/style.css]: Done

**Review by Visaali-Dentsu — APPROVED** [review]

**Review by VisaaliSuccessive — APPROVED** [review]

**Pull request merged**